src/DigitalSubscriberAppBanner.tsx

```diff
@@ -70,8 +70,8 @@ export const contentContainer = css`
 export const topLeftComponent = css`
     width: 93%;
     padding: ${space[4]}px;
-
-    ${from.desktop} {
+    min-height: ${imgHeight};
```

Comment:
Should we specify a unit here (`px`) for consistency? (I think _technically_ it's not valid css to omit the unit, might be wrong though).

Reply:
Cheers Tom, both fixed now!